README.md

```diff
@@ -50,25 +50,73 @@ You should start 4 instances parallel.
 Instance 1
 
 ```
+First run
```

Comment:
@mvbeek in the code section, can you remove the explanation? please include the command.